app/main.py

```diff
@@ -14,4 +14,6 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    low_phrase = phrase.lower()
+    result_string = low_phrase.count(letter.lower())
+    return result_string
```

Comment:
rewrite this function to one line with using method `count` of `string`

Reply:
> rewrite this function to one line with using method `count` of `string`

Done

Reply:
Should I create a new pull request ?